src/components/cartItem/index.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+
+import Button from "../common/button";
+import Image from "../common/image";
+
+import "./index.scss";
+
+import { mrpLabel, ruppeeLabel } from "../../constant";
+
+const CartItem = ({
+  item,
+  handleIncrementItem,
+  handleDecrementItem,
+  handleRemoveItem,
+}) => {
+  // increment cart item qty
+  const handleIncrementItemQty = () => {
+    handleIncrementItem && handleIncrementItem(item);
+  };
+
+  // decrement cart item qty
+  const handleDecrementItemQty = () => {
+    handleDecrementItem && handleDecrementItem(item);
+  };
+
+  // remove item from cart
+  const handleRemoveItemQty = () => {
+    handleRemoveItem && handleRemoveItem(item);
+  };
+
+  return (
+    <div className="cart-item" key={item.id}>
+      <div className="cart-item-image">
+        <Image
+          imgClassName="img-fluid"
+          src_2x={item.imageURL}
+          src={item.imageURL}
+          alt={item.name}
+        />
+      </div>
+      <div className="cart-item-info">
+        <div className="cart-item-title">{item.name}</div>
+        <div className="cart-item-calculation">
+          <Button
+            variant="primary"
+            className="button btn-rounded cart-decrement"
+            onClick={
+              item.count <= 1 ? handleRemoveItemQty : handleDecrementItemQty

```

Comment:
create a common function and pass item.count in decide, decide on the basis of it

Reply:
COmpleted.